public/locales/en-US/send.ftl

```diff
@@ -176,9 +197,10 @@ accountBenefitExpiryTwo = { $count ->
     *[other] Keep links active for up to { $count } days
 }
 accountBenefitSync = Manage shared files from any device
+accountBenefitMoz = Learn about other Mozilla services
 manageAccount = Manage Account
 logOut = Sign Out
-okButton = Ok
+okButtonUpdate = OK
```

Comment:
we don't need a new id for a case change iirc. @flodolo correct?

Reply:
No, no need